### DISCUSSION TITLE
Eksik sayı

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#45] [Beyin Fırtınası]
Ormanın kuşları bir akşam yine Bilge Karga’yı görmeye gelmişler. Kalabalığı gören Bilge Karga ne olduğunu anlamak için dışarı çıkmış. Kuşlardan birine ne istediklerini sormuş.

Kuş: Komşu ormandan bir kuş geldi, Bilge Karga. Kesin büyücü. Çok korkuyoruz.

Bilge Karga: Nasıl büyücü? Neler yapıyor?

Kuş: Beynimizi okuyor.

Bilge Karga: Sakin ol ve bunu nasıl yaptığını anlat.

Kuş: Nasıl yaptığını bilmiyoruz. Belki sen biliyorsundur diye sana geldik.

Bilge Karga: Peki, şimdi sakince bu yeni kuşun ne yaptığını anlat.

Kuş: 1 ile 100 arasında aklımızda bir sayı tutmamızı istiyor. Sonra 1’den 100’e kadar bu tuttuğumuz sayı dışındaki sayıları istediğimiz sırada kendisine söylememizi istiyor. Bunu yapıyoruz ve hemen ardından aklımızda tuttuğumuz sayıyı biliyor.

Bilge Karga: Belki söylediğiniz sayıları not alıyordur.

Kuş: Hayır, hep beraber kontrol ettik. Sadece durup söylediğimiz sayıları dinliyor, başka hiçbir hareket yapmıyor.

Bilge Karga: Peki, yarın gelin, bunu nasıl yaptığını düşüneyim ben de.

Acaba ormana yeni gelen kuş bunu nasıl yapıyordu?

**Comment by yigidi** [comment]
Bu tür zeka oyunlarında sıkça karşılaşılan bir durumla karşı karşıyasınız gibi görünüyor. İşte yeni gelen kuşun muhtemelen kullanmış olduğu yöntem:

1 ile 100 arasında herhangi bir sayıyı tahmin edin.
Sonra, bu tahmin edilen sayının, size söylenen sayıların arasında geçip geçmediğini kontrol edin.
Tahmin ettiğiniz sayı söylenen sayıların arasında yer almıyorsa, bu tahmin edilen sayı aklınızda tuttuğunuz sayıdır.
Örneğin, kuşların 1 ile 100 arasında bir sayı tahmin ettiğini düşünün ve sizin sırayla 1'den 100'e kadar sayıları söylediğinizi varsayalım. Eğer kuşun tahmin ettiği sayı sırasıyla söylenen sayıların arasında hiç geçmiyorsa, o tahmin edilen sayı kuşun aklında tuttuğu sayıdır.

Bu yöntemle kuş, tahmin ettiği sayının verilen aralıkta olup olmadığını anlayarak sizin söylediğiniz sayıları dinlerken gerçek sayısını tespit edebilir.
